charts/ibm-mq/README.md

```diff
@@ -43,6 +43,12 @@ See the **configuration** section for how to configure these values.
 
 > **Tip**: A [samples directory](../../samples/README.md) is available that demonstrates how to install the chart in a number of common environments. For first time users it is suggested that this is used.
 
+Add reposity to helm with the command:
```

Comment:
Could we change this to:
"To add the helm chart to your helm repository, run the command:"

Reply:
Changed